main.go

```diff
@@ -110,7 +113,24 @@ func main() {
 				Port:    9443,
 				TLSOpts: []func(config *tls.Config){disableHTTP2},
 			}),
-	})
+	}
+
+	if leaseDuration := getEnvInDuration("LEASE_DURATION"); leaseDuration != 0 {
+		setupLog.Info("manager configured with lease duration", "seconds", int(leaseDuration.Seconds()))
+		options.LeaseDuration = &leaseDuration
+	}
+
+	if renewDeadline := getEnvInDuration("RENEW_DEADLINE"); renewDeadline != 0 {
+		setupLog.Info("manager configured with renew deadline", "seconds", int(renewDeadline.Seconds()))
+		options.RenewDeadline = &renewDeadline
+	}
+
+	if retryPeriod := getEnvInDuration("RETRY_PERIOD"); retryPeriod != 0 {
+		setupLog.Info("manager configured with retry period", "seconds", int(retryPeriod.Seconds()))
+		options.RetryPeriod = &retryPeriod
+	}
```

Comment:
just thinking about to get this in sync across the operators, could also do this in lib-common and just pass in a ptr to `options` ? like if we'd then later add another ENV var, we'd just need to add it to lib-common.